app/locale/fr_FR/Oyst_OneClick.csv

```diff
@@ -146,3 +146,5 @@
 "Actions","Actions"
 "Delete","Effacer"
 "Download","Télécharger"
+
+"Impossible to issue a refund transaction because the capture transaction does not exist","Impossible d'émettre une transaction de remboursement parce que la transaction de capture n'existe pas"
```

Comment:
@StefBrito je te laisse valider la trad ;)

Reply:
"Impossible to initiate a refund transaction because the capture transaction does not exist"
French "Impossible de lancer le remboursement de la commande car celle ci n'a pas été capturée" 
@mehdichaouch Tu en penses quoi ?